bears/general/SpaceConsistencyBear.py

```diff
@@ -34,12 +35,55 @@ def run(self,
             Number of spaces per indentation level.
         :param enforce_newline_at_EOF:
             Whether to enforce a newline at the End Of File.
+        :param allow_leading_blanklines:
+            Whether to allow leading blank lines at the start
+            of file or not.
         '''
         spacing_helper = SpacingHelper(indent_size)
         result_texts = []
         additional_info_texts = []
 
-        for line_number, line in enumerate(file, start=1):
+        def end_blanklines():
+            end_line = False
+            enumerated_zip_obj = zip(range(1, len(file) + 1), file)
+            enumerated_tuple = tuple(enumerated_zip_obj)
+
+            for line_number, line in enumerated_tuple:
+                replacement = line
+                if replacement.strip() == '':
+                    end_line = line_number
+                else:
+                    break
+            return end_line
```

Comment:
In your case, `end_line` is either False or a number. I suggest not doing this.
I would have 2 functions, `has_leading_blanklines()` and `get_leading_blanklines_no()` or similar.